### PR TITLE
Makes phone number input return E.164 standard

### PR DESCRIPTION
**Pull request by tanatorn**
As a pre-requisite of the phone SMS login codes, I'm cleaning up the settings page with a better phone input. 

This will allow us to record full E.164 numbers, e.g.  + (206)-123-4678 instead of just free form phone numbers. This is the format that Twilio requires.

I tested that onboarding and re-editing the page works

### CODE REVIEW

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -89,4 +90,18 @@
     <%= form.submit @user.full_name.present? ? 'Save profile' : 'Start using Bank' %>
   </div>
 <% end %>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/17.0.13/js/intlTelInput.min.js" integrity="sha512-QMUqEPmhXq1f3DnAVdXvu40C8nbTgxvBGvNruP6RFacy3zWKbNTmx7rdQVVM2gkd2auCWhlPYtcW2tHwzso4SA==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/17.0.13/css/intlTelInput.css" integrity="sha512-gxWow8Mo6q6pLa1XH/CcH8JyiSDEtiwJV78E+D+QP0EVasFs8wKXq16G8CLD4CJ2SnonHr4Lm/yY2fSI2+cbmw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <script>
+      const phoneInputField = document.querySelector("#phone_raw");
```
Comment (maxwofford) [inline app/views/users/edit.html.erb]: Please keep indentation consistent.

**Review by maxwofford — APPROVED** [review]
This looks great! Thanks for adding this.

File: app/views/users/edit.html.erb [hunk 2]
```diff
@@ -89,4 +90,18 @@
...
+      const phoneInput = window.intlTelInput(phoneInputField, {
+          initialCountry: 'us',
+          utilsScript:
+              "https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/17.0.8/js/utils.js",
+      });
+      function onSubmit() {
+          document.getElementById('phone_number').value = phoneInput.getNumber();
+          return true;
+      }
+  </script>
```
Comment (garyhtou) [inline app/views/users/edit.html.erb]: Should we consider moving this into `app/assets/javascripts/` and using `javascript_include_tag`?

**Review by garyhtou — APPROVED** [review]

File: app/views/users/edit.html.erb (hunk 2 above)
Comment (tanatorn) [reply app/views/users/edit.html.erb]: good idea :)

**Pull request merged**